pkg/tests/api/alerting/api_alertmanager_test.go

```diff
@@ -842,24 +841,8 @@ func TestIntegrationDeleteFolderWithRules(t *testing.T) {
 		_, err = io.ReadAll(resp.Body)
 		require.NoError(t, err)
 		require.Equal(t, 200, resp.StatusCode)
-	}
-
-	// Finally, we ensure the rules were deleted.
-	{
-		u := fmt.Sprintf("http://editor:editor@%s/api/ruler/grafana/api/v1/rules", grafanaListedAddr)
-		// nolint:gosec
-		resp, err := http.Get(u)
-		require.NoError(t, err)
-		t.Cleanup(func() {
-			err := resp.Body.Close()
-			require.NoError(t, err)
-		})
-		b, err := io.ReadAll(resp.Body)
-		require.NoError(t, err)
-
-		assert.Equal(t, 200, resp.StatusCode)
-		assert.JSONEq(t, "{}", string(b))
-	}
```

Comment:
I think you forgot to add this back.

Reply:
true! I added it as a separate test case. Thanks for noticing!